include/swift/RemoteInspection/TypeRefBuilder.h

```diff
@@ -400,6 +401,10 @@ class TypeRefBuilder {
   TypeRefBuilder(const TypeRefBuilder &other) = delete;
   TypeRefBuilder &operator=(const TypeRefBuilder &other) = delete;
 
+  Mangle::ManglingFlavor getManglingFlavor() {
+    return Mangle::ManglingFlavor::Default;
```

Comment:
Can you make this a parameter in the constructor? It can be defaulted to true. Reason being that LLDB uses TypeRefBuilder to process types found in DWARF.